app/adminPage.html

```diff
@@ -50,5 +62,12 @@ <h2>Scores</h2>
         </div>
     </div>
 </section>
+<div class="container-fluid logo-bottom">
+    <div class="row">
+        <div class="col-xs-12">
+            <img class="mayden-logo" src="images/logo-mayden.svg">
+        </div>
```

Comment:
If there are scores present and the new user form is opened the scores are pushed into and clip with the Mayden logo.